templates/webserver/webserver-service.yaml

```
@@ -25,3 +25,10 @@ spec:
       protocol: TCP
       port: {{ .Values.ports.airflowUI }}
       targetPort: {{ .Values.ports.airflowUI }}
+    {{- if .Values.authSidecar.enabled }}
+    - name: nginx-ui
+      protocol: TCP
+      port: 8084
```

And if we did that (above) we could to listen on the normal port, `{{ .Values.ports.airflowUI }}`. That'd allow the use of the same port regardless of whether the auth sidecar was in use or not.
```suggestion
      port: {{ .Values.ports.airflowUI }}
```

---

@pgvishnuram ☝️

---

different ports were used because of the use of authentication, airflow health check will fail if we use same port.

---

What health check? liveness/readiness probes are at the container level, not through the service. Is there something else doing health checks in addition?

Even still, /health should probably be exempt from auth requirements.

---

Either way, when we transition to being based on the OSS chart I think we will just be adding new ports, so I'm just going to resolve this.